docs/user/build-customization.rst

```diff
@@ -104,6 +105,43 @@ To avoid this, it's possible to unshallow the clone done by Read the Docs:
          - git fetch --unshallow
 
 
+Skip build based on a condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There may be situations where you want to skip a build that was automatically triggered when someone on your team pushed to the repository.
+Skipping builds will allow you to speed up review times and also help us to reduce costs in our servers.
+The following situations are good examples to skip a build:
+
+* the build depends on an external situation that's not met yet
+* there were no changes on the documentation files
+
+In these scenarios, you can skip the build by executing a custom command that checks for that particular condition
+and exits with code ``439`` to skip it, or ``0`` to continue building the documentation normally.
+If any of the commands return this particular exit code,
+Read the Docs will stop the build immediately,
+mark it as "Cancelled",
+and communicate GitHub/GitLab that the build succeeded (green tick) so the pull request is in a mergeable state.
```

Comment:
```suggestion
and communicate to your Git platform (GitHub/GitLab/Bitbucket) that the build succeeded (green tick ✅) so the pull request is in a mergeable state.
```

Reply:
```suggestion
and communicate to your Git platform (GitHub/GitLab) that the build succeeded (green tick ✅) so the pull request is in a mergeable state.
```

I updated the suggestion because we don't support PR builders on Bitbucket.